components/homme/src/theta-l/element_ops.F90

```diff
@@ -483,10 +483,10 @@ subroutine get_state(u,v,w,T,pnh,dp,ps,rho,zm,zi,g,elem,hvcoord,nt,ntQ)
     u   = elem%state%v   (:,:,1,:,nt)
     v   = elem%state%v   (:,:,2,:,nt)
     ps  = elem%state%ps_v(:,:,  nt)
```

Comment:
Should this ps variable be removed then? Is it correct that ps_v should only appear in homme standalone init and in cam inits?